workers/unity/Packages/com.improbable.gdk.core/Systems/ComponentSendSystem.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Improbable.Gdk.Core.CodegenAdapters;
+using Improbable.Worker.CInterop;
+using Unity.Entities;
+using UnityEngine.Profiling;
+
+namespace Improbable.Gdk.Core
+{
+    /// <summary>
+    ///     Executes the default replication logic for each SpatialOS component.
+    /// </summary>
+    [DisableAutoCreation]
+    [AlwaysUpdateSystem]
+    [UpdateInGroup(typeof(SpatialOSSendGroup.InternalSpatialOSSendGroup))]
+    [UpdateBefore(typeof(ComponentUpdateSystem))]
+    public class ComponentSendSystem : ComponentSystem
+    {
+        // Can't access the generated component ID in Core code.
+        private const uint PositionComponentId = 54;
+
+        private Connection connection;
+
+        private readonly List<ComponentReplicator> componentReplicators =
```

Comment:
super nit: move the readonly variable before the connection variable

Reply:
Ah yep. This was copy pasted from the old ones. Will move it.